images/nginx/build.sh

```diff
@@ -173,9 +173,6 @@ get_src 1ad2e34b111c802f9d0cdf019e986909123237a28c746b21295b63c9e785d9c3 \
         "http://luajit.org/download/LuaJIT-2.1.0-beta3.tar.gz"
 
 
-#https://blog.cloudflare.com/optimizing-tls-over-tcp-to-reduce-latency/
-curl -sSL -o nginx__dynamic_tls_records.patch https://raw.githubusercontent.com/cloudflare/sslconfig/master/patches/nginx__1.11.5_dynamic_tls_records.patch
```

Comment:
why are you removing this?

Reply:
yep, let's sync the master with upstream then

Reply:
Master is now synced with upstream master. Github does not update the diff on the PR when the target branch get's updated. We would have to close this PR and create another PR.